scripts/cmake/ExternalProjectCatalyst.cmake

```diff
@@ -9,6 +9,11 @@ endif()
 
 # CLI modules
 set(PARAVIEW_MODULES vtkIOXML)
+### The following if-condition is not used now but future,
+### and it is commented out and left here just as a reminder. 
+#if(OGS_USE_MPI OR OGS_USE_PETSC)
+#	set(PARAVIEW_MODULES ${PARAVIEW_MODULES} vtkIOParallelXML)
+#endif()
```

Comment:
With this block commented out the compilation fails with undefined reference to vtk...:
```
../lib/libFileIO.a(VtuInterface.cpp.o): In function `vtkSmartPointer<vtkXMLPUnstructuredGridWriter>::New()':
/usr/include/vtk/vtkSmartPointer.h:117: undefined reference to `vtkXMLPUnstructuredGridWriter::New()'
clang-3.7: error: linker command failed with exit code 1 (use -v to see invocation)
Tests/CMakeFiles/testrunner.dir/build.make:2357: recipe for target 'bin/testrunner' failed
make[2]: *** [bin/testrunner] Error 1
```